docs/api/api/most_active_packages.xsd

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
+
+  <xs:element name="most_active_packages">
+    <xs:annotation>
+      <xs:documentation>
+        List of most active packages or projects in the openSUSE build
+        service ordered activity.
+      </xs:documentation>
+    </xs:annotation>
+    <xs:complexType>
```

Comment:
I think this need to be fixed. We only have one type here now.

Reply:
Hi @perlpunk, this is because you need to adapt the tests, since you added 2 new schemas and removed `most_active`. Have a look [here](https://github.com/openSUSE/open-build-service/blob/master/src/api/test/functional/backend_test.rb#L18).

Reply:
thanks, I had missed that one when grepping the sources :)